bot/extensions/information.py

```diff
@@ -194,3 +194,45 @@ async def platforms(self, ctx: custom.Context) -> None:
                             "● **Vimeo** *(Links)*\n"
             )
         )
+
+    @commands.group(name="prefix", invoke_withour_command=True)
+    async def _prefix(self, ctx: custom.Context):
+
+        if ctx.invoked_subcommand:
```

Comment:
I assume this is just because you couldn't figure out why the root command was invoking as well? if not then no worries!

Reply:
Nah I do this one personally. I can remove if you like.